website/lab/05/index.mdx

```diff
@@ -510,28 +510,42 @@ Because writing an SPI driver for the ST7735 would be out of the scope of the la
 use embassy_embedded_hal::shared_bus::blocking::spi::SpiDevice;
 use embassy_sync::blocking_mutex::{Mutex, raw::NoopRawMutex};
 use embassy_time::Delay;
-use display_interface_spi::SPIInterface;
+use mipidsi::interface::SpiInterface;
 use mipidsi::models::ST7735s;
 use mipidsi::options::{Orientation, Rotation};
 
-
-let mut screen_config = embassy_stm32::spi::Config::default();
-screen_config.frequency = Hertz(3_000_000);
+let mut config = Config::default();
```

Comment:
I think we would maybe explain this in a tip. This is actually necessary to make the screen drawing faster, without the it works, but it is slow.